test/extended/builds/pipeline_origin_bld.go

```diff
@@ -385,24 +391,36 @@ var _ = g.Describe("[Feature:Builds][Slow] openshift pipeline build", func() {
 				// NOTE, for the credential URL in Jenkins
 				// it returns rc 200 with no output if credential exists and a 404 if it does not exists
 				_, err = j.WaitForContent("", 200, 10*time.Second, "credentials/store/system/domain/_/credential/%s-%s/", oc.Namespace(), secretName)
+				if err != nil {
+					exutil.DumpApplicationPodLogs("jenkins", oc)
+				}
```

Comment:
Why not add this to the `AfterEach` block at the start of the suite? See https://github.com/openshift/origin/blob/7d8731afc7213cda40cdfe0741cbf7723472a1a3/test/extended/builds/pipeline_origin_bld.go#L63-L67

Reply:
presumably the `exutil.DumpPodLogsStartingWith("", oc) ` should have caught it 

i suspect the pod is getting deleted now before we hit this path

Reply:
@gabemontero looks like this test isn't using the Ginkgo `BeforeEach`/`AfterEach` conventions. How hard would it be to refactor the `cleanup` and `debugAnyJenkinsFailure` methods to this?